specs/lib/removeUnusedCamerasSpec.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var removeUnusedCameras = require('../../').removeUnusedCameras;
+var OptimizationStatistics = require('../../').OptimizationStatistics;
+
+describe('removeUnusedCameras', function() {
+    it('removes a camera', function() {
+        var gltf = {
+            "cameras": {
+                "camera_0": {
+                    "perspective": {
+                        "aspect_ratio": 1.5,
+                        "yfov": 0.660593,
+                        "zfar": 100,
+                        "znear": 0.01
+                    },
+                    "type": "perspective"
+                },
+                "unusedCameraId": {
+                    "perspective": {
+                        "aspect_ratio": 1.5,
+                        "yfov": 0.660593,
+                        "zfar": 100,
+                        "znear": 0.01
+                    },
+                    "type": "perspective"
+                }
+            },
+            "nodes": {
+                "node_3": {
+                    "camera": "camera_0",
+                    "children": [

```

Comment:
`children`, `matrix`, and `name` are optional so let's remove them to keep this simple.

See https://github.com/KhronosGroup/glTF/blob/master/specification/README.md#node